frontend/src/lib/forms/UserTypeahead.svelte

```diff
@@ -23,6 +24,8 @@
     [],
     debounceMs);
 
+  $: filteredResults = $typeaheadResults.filter(user => !exclude.includes(user.id));
```

Comment:
I like that this is in the frontend; it lets `usersICanSee` do exactly what it says on the <del>tin</del> function signature.